src/components/dialogmoteikkeaktuell/DialogmoteikkeaktuellSkjemaContainer.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Side from "@/sider/Side";
+import SideLaster from "@/components/SideLaster";
+import Sidetopp from "@/components/Sidetopp";
+import DialogmoteikkeaktuellSkjema from "@/components/dialogmoteikkeaktuell/DialogmoteikkeaktuellSkjema";
+import { Menypunkter } from "@/components/globalnavigasjon/GlobalNavigasjon";
+
+const texts = {
+  pageTitle: "Ikke aktuell",
+};
+
+const DialogmoteikkeaktuellSkjemaContainer = () => {
+  return (
+    <Side tittel={texts.pageTitle} aktivtMenypunkt={Menypunkter.DIALOGMOTE}>
+      <SideLaster henter={false} hentingFeilet={false}>
+        <Sidetopp tittel={texts.pageTitle} />
+        <DialogmoteikkeaktuellSkjema />
+      </SideLaster>
+    </Side>
```

Comment:
Synes vi bør tredele siden her slik vi har gjort på `DialogmoteInnkallingContainer` 😊 For å få inn historikk for møte og møtesvar. Grunnen til at vi tok den inn i tredelingen her var at historisk info var relevant når de skulle kalle inn. Har en antakelse om at det er relevant også her.

![Screenshot 2024-06-10 at 16 50 41](https://github.com/navikt/syfomodiaperson/assets/11747383/6f6ef33d-ded7-489b-aabb-bbb869b132d8)

Reply:
Ja, det virker lurt. Men som en egen PR da?

Reply:
Trenger ikke å være en egen PR for min del. Er gjort ganske enkelt i `DialogmoteInnkallingContainer`. Gjør det du tenker er best 😊

Reply:
OK, tror jeg fikk det til.